libcxx/include/__hash_table

```diff
@@ -775,6 +779,7 @@ public:
   }
 
 private:
+  _LIBCPP_HIDE_FROM_ABI __next_pointer __node_insert_multi_find_insertion_point(size_t __cp_hash, value_type& __cp_val);
```

Comment:
`__node_insert_multi_find_insertion_point` is used in `__node_insert_multi_prepare` and `__debug_randomize_order`. Is refactoring warranted or should I instead copy the "find insertion point" logic/code from `__node_insert_multi_prepare`?

Reply:
I like that you extracted this into a separate function.